pkg/client/clientset/versioned/typed/controlplane/v1beta2/clustergroupmembers.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Code generated by client-gen. DO NOT EDIT.
+
+package v1beta2
+
+import (
+	"context"
+
+	v1beta2 "github.com/vmware-tanzu/antrea/pkg/apis/controlplane/v1beta2"
+	scheme "github.com/vmware-tanzu/antrea/pkg/client/clientset/versioned/scheme"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	rest "k8s.io/client-go/rest"
+)
+
+// ClusterGroupMembersesGetter has a method to return a ClusterGroupMembersInterface.
+// A group's client should implement this interface.
+type ClusterGroupMembersesGetter interface {
+	ClusterGroupMemberses() ClusterGroupMembersInterface
```

Comment:
You can specify "--plural-exceptions" when calling client-gen to avoid it

Reply:
Thanks for coming for the rescue, as always :) Updated